fast_api_als/routers/submit_lead.py

```diff
@@ -123,35 +136,45 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
 
     # if dealer is not available then find nearest dealer
     if not dealer_available:
+        logging.info("Dealer not available, finding nearest dealer")
         lat, lon = get_customer_coordinate(obj['adf']['prospect']['customer']['contact']['address']['postalcode'])
         nearest_vendor = db_helper_session.fetch_nearest_dealer(oem=make,
                                                                 lat=lat,
                                                                 lon=lon)
         obj['adf']['prospect']['vendor'] = nearest_vendor
         dealer_available = True if nearest_vendor != {} else False
+        if(dealer_available): logging.info("Dealer found")
+        else: logging.info("Dealer not found")
 
     # enrich the lead
+    logging.info("Enriching the lead")
     model_input = get_enriched_lead_json(obj)
 
     # convert the enriched lead to ML input format
+    logging.info("Converting to ML input format")
     ml_input = conversion_to_ml_input(model_input, make, dealer_available)
```

Comment:
do exceptions need to be handled here?

Reply:
These log messages are to log time stamps for each of this processing steps for analysis